static/js/lib/notificationsApi.js

```diff
@@ -44,6 +45,11 @@ export function parseStoredUserMessage(
       SETTINGS.support_email
     }.`
     break
+  case USER_MSG_TYPE_ENROLL_BLOCKED:
+    alertType = ALERT_TYPE_DANGER
+    msgText =
```

Comment:
Once this message is shown to the user upon clicking the `Enroll now` button, it doesn't fade away automatically nor does it hide when the user navigates to some other page e.g. `HomePage` until after a few seconds which suppose is the default interval for the message.

I think we should reduce the interval in this case.

Reply:
I set the expiration to 20 sec to account for the potential of a slow API response. I'm going to create a separate issue to hide the message if it has already been viewed